content/docs/command-prompt.md

```diff
@@ -200,6 +200,12 @@ The Notepad++ [installer executable](../getting-started/#installer) accepts the
 - `/D=c:\blah` or `/D=c:\path with spaces\blah` : Overrides the default installation directory.
     - Do _not_ put quotes around the path, even when there are spaces.
     - Because it allows spaces in the path, this option **must** be the last argument on the installer command line, if included.
+- `/closeRunningNpp`: Will (try to) close existing Notepad++ before installing the new version.  (New to v8.6.9.)
+    - It will first try a "nice" close request (using [WM_CLOSE](https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-close)).
+    - If the "nice" request wasn't successful after 5 seconds, it will kill the underlying process.
+    - If Notepad++ has multiple instances opened, it will close all instances.
+- `/runNppAfterSilentInstall`: After a silent install, it will automatically run the newly-installed Notepad++. (New to v8.6.9.)
+    - Only works if `/S` is also specified.

```

Comment:
@pryrt 

I think that these two new options belongs to the next "It also implements an additional option:"  (and it should be updated to `It also implements additional options:`) part and not to the "The Notepad++ installer executable accepts the `three` NSIS command-line options:"

And the [installer executable](https://github.com/notepad-plus-plus/npp-usermanual/blob/18d2bd6808c1a69c3e8c09440635e5a724adf7a5/content/getting-started/#installer) link there leads to `404 - page not found`